requirements.txt

```diff
@@ -0,0 +1 @@
+Sphinx
```

Comment:
Is it possible to move this requirement document to `doc`? So that it will be clear that it's requirements only for Documentation, not for the full oneMKL build.